bin/check-rabbitmq-amqp-alive.rb

```diff
@@ -74,15 +90,21 @@ def run
   end
 
   def vhost_alive?
-    host     = config[:host]
-    port     = config[:port]
-    username = config[:username]
-    password = config[:password]
-    vhost    = config[:vhost]
-    ssl      = config[:ssl]
+    host           = config[:host]
+    port           = config[:port]
+    username       = config[:username]
+    password       = config[:password]
+    vhost          = config[:vhost]
+    ssl            = config[:ssl]
```

Comment:
@analytically it mostly depends on the setup. If you do not have verify_peer or fail_if_no_peer_cert enabled on the server, and you disable peer verification on the client, just toggling `ssl` is sufficient, I believe.